hathorlib/utils.py

```diff
@@ -180,3 +183,15 @@ def get_hash160(public_key_bytes: bytes) -> bytes:
     h = hashlib.new('ripemd160')
     h.update(key_hash.digest())
     return h.digest()
+
+
+def is_nft_creation(tx: 'BaseTransaction') -> bool:
+    """Returns True if the transaction is an NFT creation"""
+    from hathorlib import TokenCreationTransaction
+    from hathorlib.base_transaction import TxVersion
+    if tx.version != TxVersion.TOKEN_CREATION_TRANSACTION:
```

Comment:
If we change the method's signature, I feel we should raise an exception here (or just change it to an assert).